src/equisolve/utils/convert.py

```diff
@@ -35,17 +36,27 @@ def ase_to_tensormap(
     if not isinstance(frames, list):
         frames = [frames]
 
-    values = [f.info[energy] for f in frames]
+    if energy is not None:
+        values = [f.info[energy] for f in frames]
+    else:
+        energy = "energy"
+        values = [f.get_potential_energy() for f in frames]
```

Comment:
this also needs a check if energy is present right?

Reply:
mhh, honestly I wanted this to fail since having gradients available but not the property, seems a bit weird to me

Reply:
you are right, forgot that this just extracts the property